kotlin/workflow-runtime/src/main/java/com/squareup/workflow/internal/Workers.kt

```diff
@@ -37,9 +38,14 @@ import kotlinx.coroutines.selects.SelectBuilder
 @UseExperimental(FlowPreview::class, ExperimentalCoroutinesApi::class)
 internal fun <T> CoroutineScope.launchWorker(worker: Worker<T>): ReceiveChannel<Output<T>> =
   worker.run()
-      // TODO(https://github.com/square/workflow/issues/434) Remove this map to allow operator
-      // fusion to occur.
       .map { Output(it) }
```

Comment:
You removed the TODO. Did something fix fusion for us?

Reply:
No, the `catch` operator means we can't get fusion anymore.